src/wcag-taxonomy-provider.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import * as Sarif from './sarif/sarif-2.1.2';
+import { axeTagsToWcagLinkData, WCAGLinkData } from './wcag-link-data';
+
+export function getWcagTaxonomy(sortedWcagTags: string[]): Sarif.ToolComponent {
+    return {
+        name: 'WCAG',
+        fullName: 'Web Content Accessibility Guidelines (WCAG) 2.1',
+        organization: 'W3C',
+        informationUri: 'https://www.w3.org/TR/WCAG21',
+        version: '2.1',
+        guid: '',
+        isComprehensive: true,
+        taxa: getAllTaxaFromWcagLinkData(sortedWcagTags),
+    };
+}
+
+function getAllTaxaFromWcagLinkData(
+    sortedWcagTags: string[],
+): Sarif.ToolComponent['taxa'] {
+    var taxa: Sarif.ToolComponent['taxa'] = [];
```

Comment:
This function body is mostly re-writing the Array.map implementation; consider instead:

```
return sortedWcagTags.map(tag =>
    getIndividualTaxaFromWcagLinkData(tag, axeTagsToWcagLinkData[tag]));
```